content/en/docs/getting-started/_index.html

```diff
@@ -0,0 +1,266 @@
+  <div class="card-body">
+    <h4 class="card-title"><strong>Did you know?</strong></h4>
+    <p class="card-text">There are other ways to trial and run Jenkins X in production that have been developed from the Jenkins X open source project. Learn more about these under <a a class="card-link" href="/docs/alternatives"><strong>Jenkins X OSS alternatives</strong></a></p>
+  </div>
+</div>
+
+  <p>&nbsp;</p>
+  <div class="row">
+    <div class="col col-lg-12">
+      <h2>Step 1: Install the Jenkins X CLI</h2>
+      <p>
+        The first step in using Jenkins X is installing the Jenkins X binary <strong>jx</strong> for
+        your operating system.
+      </p>
+      <ul class="nav nav-tabs" id="install-jx-tabs">
+        <li class="nav-item">
+          <a class="nav-link active" data-toggle="tab" href="#osx">OS X</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" data-toggle="tab" href="#linux">Linux</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" data-toggle="tab" href="#windows">Windows</a>
+        </li>
+      </ul>
+      <div class="tab-content" id="install-jx-tabs">
+        <div class="tab-pane fade show active" id="osx">
+        <h3>Using Homebrew</h3>
+          
+{{<highlight sh>}}
+brew install jenkins-x/jx/jx
+{{</highlight>}}
+          
+        <h3>Using Curl</h3>
+        <p>
+        Download the `.tar` file, unarchive it.
+        </p>
+
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-darwin-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+        
+        
+        Add  `jx` to your path
+        
+{{<highlight sh>}}
+sudo mv jx /usr/local/bin
+{{</highlight>}}
+        
+        Check if all is good!
+        
+{{<highlight sh>}}
+jx --version
+{{</highlight>}}
+        </div>
+        <div class="tab-pane fade" id="linux">
+        
+        &nbsp;  
+        Download the `.tar` file, and unarchive it in a directory where you can run the `jx` command.
+      
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent https://api.github.com/repos/jenkins-x/jx/releases/latest | jq -r '.tag_name')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+      
+        or, if you don't have `jq` installed:
+      
+{{<highlight sh>}}
+curl -L "https://github.com/jenkins-x/jx/releases/download/$(curl --silent "https://github.com/jenkins-x/jx/releases/latest" | sed 's#.*tag/\(.*\)\".*#\1#')/jx-linux-amd64.tar.gz" | tar xzv "jx"
+{{</highlight>}}
+      
+        Move binary to your path
+      
+{{<highlight sh>}}
+sudo mv jx /usr/local/bin
+{{</highlight>}}
+      
+      Test all is good
+      
+{{<highlight sh>}}
+jx --version
+{{</highlight>}}
+      
+        </div>
+        <div class="tab-pane fade" id="windows">
+        &nbsp;
+      
+        <h3> Install Chocolatey using an Admin</h3>
```

Comment:
Why not just explicitly document the `curl` approach and then just mention that `jx` is published via Chocolatey and Scoop?

Reply:
I agree. I believe the Chocolatey and Scoop were there as legacy just as the Homebrew option used to be there for macOS before we removed it in CJXD. Are the Chocolatey and Scoop packages maintained within CB or at least by a person connected to the OSS project?

Reply:
I think we still deploy to Chocolatey, but that does not mean we have to fully document everything. One way which works would be good enough imo.